certauth.go

```diff
@@ -197,22 +238,31 @@ func (a *Auth) CheckAuthorization(
 		err    error
 	)
 
-	for _, ck := range a.opt.AuthorizationCheckers {
-		if ps == nil { // not using httprouter
-			params, err = ck.CheckAuthorization(ou, cn)
-		} else { // using httprouter
-			params, err = ck.CheckAuthorizationWithParams(ou, cn, ps)
-		}
-		if err != nil {
-			return nil, err
-		}
-
-		// Collect the context params from each AuthorizationChecker into one map
-		if params != nil {
-			for k, v := range params {
-				ctxParams[k] = v
+	checkers := append([][]AuthorizationChecker{}, a.checkers...)
+	if len(a.opt.AuthorizationCheckers) > 0 {
+		checkers = append(checkers, a.opt.AuthorizationCheckers)
+	}
```

Comment:
It seems unnecessary to do this each time you run the function, as the content will always be the same. But I don't feel very strongly about it so :shrug: